README.md

```diff
@@ -1,219 +1,268 @@
+   `appInsights.start();`.
+
+[these instructions]: https://azure.microsoft.com/documentation/articles/app-insights-create-new-resource/
+
+
+## Basic Usage
+
+To track HTTP requests, unhandled exceptions and system metrics:
 
 ```javascript
-import appInsights = require("applicationinsights");
-appInsights.setup("<instrumentation_key>").start();
+let appInsights = require("applicationinsights");
+appInsights.setup("_your_ikey_").start();
 ```
 
->The instrumentation key can also be set in the environment variable APPINSIGHTS_INSTRUMENTATIONKEY. If this is done, no argument is required when calling `appInsights.setup()` or `appInsights.getClient()`.
+* If the instrumentation key is set in the environment variable
+  APPINSIGHTS\_INSTRUMENTATIONKEY, `.setup()` can be called with no
+  arguments. This makes it easy to use different ikeys for different
+  environments.
 
+Load the Application Insights library (i.e. `require("applicationinsights")`) as
+early as possible in your scripts, before loading other packages. This is needed
+so that the Application Insights libary can prepare later packages for tracking.
+If you encounter conflicts with other libraries doing similar preparation, try
+loading the Application Insights library after those.
 
+Because of the way JavaScript handles callbacks, additional work is necessary to
+track a request across external dependencies and later callbacks. By default
+this additional tracking is disabled; enable it by calling
+`appInsights.setAutoDependencyCorrelation(true)` as described in the
+Configuration section below.
 
-## Customized Usage ##
 
-### Enabling automatic correlation
+## Configuration
 
-```javascript
-import appInsights = require("applicationinsights");
-appInsights.setup("<instrumentation_key>")
-    .setAutoDependencyCorrelation(true)
-    // no telemetry will be sent until .start() is called
-    .start();
-```
+The appInsights object provides a number of configuration methods. They are
+listed in the following snippet with their default values.
 
-> Be sure to call `require("applicationinsights")` before your other imports. This allows the SDK to do patching necessary for tracking correlation state before other libraries use patched methods. If you encounter conflicts with other libraries doing similar patching, place this import below those libraries.
-
-### Disabling automatic collection
+To correlate events in a request across callbacks and dependencies, you'll need
+to call `.setAutoDependencyCorrelation(true)`.
 
 ```javascript
-import appInsights = require("applicationinsights");
+let appInsights = require("applicationinsights");
 appInsights.setup("<instrumentation_key>")
-    .setAutoCollectRequests(false)
-    .setAutoCollectPerformance(false)
-    .setAutoCollectExceptions(false)
-    .setAutoCollectDependencies(false)
-    // no telemetry will be sent until .start() is called
+    .setAutoDependencyCorrelation(false)

```

Comment:
@OsvaldoRosado set it up that way originally, discussion [here](https://github.com/Microsoft/ApplicationInsights-node.js/pull/235#issuecomment-299582120) on defaulting to true beginning in 0.20. I'll update this based on outcome there, in the meantime will note that this is experimental.